CHANGELOG.md

```diff
@@ -4,19 +4,19 @@ Mapbox welcomes participation and contributions from everyone.
 
 ### v4.9.0-alpha.1 - September 4, 2019
 
-- Fixed up bintray publish script, generate sources and javadoc jars #1065  
-- Keep *.java files in source.jar artifacts #1064
-- Added setup for bintray release process #1037
-- Add an API to check if a custom shifter is set. #1062
-- Added new nearestPointOnLine default and custom distance unit #1058
-- Adjustments to TurfMeasurement#bboxPolygon() to return a Feature instead of Polygon #1055
-- Added fuzzyMatch support to MapboxGeocoding #1051
-- Add Interceptor support to Directions Refresh API and enable Debug logging #1039
-- Add square() api to TurfMeasurement #1019
-- Adjusted link to Turf installation instructions #1038
-- Removed the last updated line from ported Turf method list #1032
-- Making GeoJson extend Serializable #1034
-- Add support for Turf envelope #1020
+- Fixed up bintray publish script, generate sources and javadoc jars [#1065](https://github.com/mapbox/mapbox-java/pull/65)
```

Comment:
The url should be `https://github.com/mapbox/mapbox-java/pull/1065`

Reply:
whoops. Fixed and rebased.